cartridge/shop/models.py

```diff
@@ -129,6 +141,8 @@ def save(self, *args, **kwargs):
         if updating and not settings.SHOP_USE_VARIATIONS:
             default = self.variations.get(default=True)
             self.copy_price_fields_to(default)
+        elif hasattr(self, 'content_model'):
+            self.content_model = self._meta.object_name.lower()
```

Comment:
For un-subclassed products, I think the value of `content_model` should be `None` rather than `'product'`. I suppose any code to that effect should be in `ContentTyped` rather than `Product`?